README.md

```diff
@@ -50,6 +50,8 @@ changelist can be configured from two sources, in order of precedence:
 If a configuration option is not specified in either file above, changelist
 falls back to the following configuration:
 
+<!--- begin default_config.toml --->
```

Comment:
```suggestion
<!--- Changes to the following block are overridden by a pre-commit hook! --->
<!--- begin default_config.toml --->
```

Reply:
I was planning to extend this to the other sections I mentioned in the first comment on the PR, which is why I made it easy to identify the section by the html comments.

Reply:
I am not following? It's a great idea to use HTML comments. But I'd include a warning above each block that changes within get overwritten.

Reply:
How about
```
<!--- begin default_config.toml overridden by a pre-commit hook! --->
```
as the start of the section. The important thing is to be able to identify what section is being started. Otherwise, when we do the same thing for 
```
.github/workflows/label-check.yaml
.github/workflows/milestone-merged-prs.yaml
```
it will be harder to tell the sections apart.

Reply:
Just in case you misunderstand, I am not implying to replace 
`<!--- begin default_config.toml --->`, but to add another comment before. I don't understand how that makes harder to tell the sections apart.

But I am happy with 
```
<!--- begin default_config.toml overridden by a pre-commit hook! --->
```
:+1:

Reply:
Sorry, I was misunderstanding. I prefer your suggestion. Thanks for being patient with me.